docs/DeveloperGuide.md

```diff
@@ -1,38 +1,269 @@
-# Developer Guide
+<div align="center">
+<h2> MoneyMind Developer Guide</h2>
+<h3> "Mind your Money" </h3>
+</div>
+
+<div align="left">
 
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Design
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Architecture
+
+<img src="images\ArchitectureDiagram.png" width="300" />
+
```

Comment:
General architecture provides a relevant overview of how the main classes are connected in this program. Well done.
![image](https://user-images.githubusercontent.com/74821808/228727576-a57da5d4-c55d-48b9-baf1-29cab200a121.png)